library/Zend/Amf/Server.php

```diff
@@ -294,10 +297,13 @@ protected function getLoader()
      * Loads a remote class or method and executes the function and returns
      * the result
      *
-     * @param  string $method Is the method to execute
-     * @param  mixed $param values for the method
+     * @param string $method Is the method to execute
+     * @param null $params
```

Comment:
Params is mixed nullable with a default of null, same for source. I would put it as mixed or mixed|null

Reply:
Though i agree that declaring nullability explicitly might be helpful, to my understanding, null is one of the types included in "mixed". 
See: https://wiki.php.net/rfc/mixed-typehint

Reply:
Fair enough, feel free to change it to just mixed then